pkg/logging/slog.go

```diff
@@ -110,6 +120,11 @@ func replaceLevelAndDropTime(groups []string, a slog.Attr) slog.Attr {
 			Key:   a.Key,
 			Value: slog.StringValue(strings.ToLower(a.Value.String())),
 		}
+	case logrErrorKey:
```

Comment:
the name of this function (`replaceLevelAndDropTime`) is no longer correct (probably already before this change).